### PR TITLE
DOC: Transfer the wiki 3rd party apps page to Markdown documentation

### PR DESCRIPTION
**Pull request by jhlegarreta**
Transfer the wiki 3rd party applications page to Markdown documentation: https://itk.org/Wiki/ITK/Third_Party_Applications.

Update the links and explanations as necessary.

Remove the mention of ITK packages for Ubuntu, as they no longer exist.

## PR Checklist
- [X] No [API changes](https://github.com/InsightSoftwareConsortium/ITK/blob/master/CONTRIBUTING.md#breaking-changes) were made (or the changes have been approved)
- [X] No [major design changes](https://github.com/InsightSoftwareConsortium/ITK/blob/master/CONTRIBUTING.md#design-changes) were made (or the changes have been approved)

### CODE REVIEW

**Comment by jhlegarreta** [comment]
Note that I removed
```
- [**Ubuntu ITK**](http://www.paulnovo.org/repository) ITK packages
  for Ubuntu (includes compiled wrappings)
```

I ignore what this was, but I do not think they are relevant now.

Other wiki pages mentioned that need to be transitioned:
https://itk.org/Wiki/ITK/Insight_Applications
https://itk.org/Wiki/ITK/CommandLineTools

File: Documentation/docs/learn/third_party_applications.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
+# Third-party applications
+
+This page lists all the applications that are known to use the Insight
```
Comment (thewtex) [inline Documentation/docs/learn/third_party_applications.md:3]: all -> some

File: Documentation/docs/learn/third_party_applications.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+Toolkit. Some of these applications are not maintained, but are kept in
+this list for informative and historical purposes.
+
+Please, add your own project details if your application uses ITK in
```
Comment (thewtex) [inline Documentation/docs/learn/third_party_applications.md:7]:
```suggestion
Please, [add your own project](../README.md) details if your application uses ITK in
```

File: Documentation/docs/learn/third_party_applications.md [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+  an open-source framework for efficient development of research tools
+  and clinical prototypes.
+- [**MARVIN**](https://www.sciencedirect.com/science/article/abs/pii/S0169260708000965)
+  was a C++ application framework for rapid and simple development of medical
+  applications, part of [NCCR "Co-Me - Computer Aided and Image Guided Medical Interventions"](https://www.snf.ch/en/9ihBpawyw4Fro4Y9/page/researchinFocus/nccr/nccr-co-me/).
+- [**MITK**](https://www.mitk.org/wiki/The_Medical_Imaging_Interaction_Toolkit_(MITK))
+  is a free open-source software for the development of interactive
+  medical image processing software.
+- [**IGSTK**](https://github.com/Kitware/IGSTK) was a free open-source
+  toolkit for development of image guided interventions software.
```
Comment (thewtex) [inline Documentation/docs/learn/third_party_applications.md:81]: No longer maintained, should be removed.

File: Documentation/docs/learn/third_party_applications.md [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+- [**Orfeo Toolbox**](https://www.orfeo-toolbox.org/) is an open-source
+  project for state-of-the-art remote sensing.
+
+### Packages
+
+- [**Python(x,y)**](https://python-xy.github.io/) is a
+  scientific-oriented Python distribution, including ITK, VTK, and Qt.
```
Comment (thewtex) [inline Documentation/docs/learn/third_party_applications.md:88]:
```suggestion
- [ITK-Wasm](https://wasm.itk.org) combines ITK and [WebAssembly](https://webassembly.org/) to enable high-performance spatial analysis in a web browser or system-level environments and reproducible execution across programming languages and hardware architectures.
- [SimpleITK](https://simpleitk.org) provides a simplified interface to ITK in a variety of languages.
```

Comment (thewtex) [reply Documentation/docs/learn/third_party_applications.md:88]: Python-XY is not maintained and there are now standard packages on PyPI for itk, vtk, qt.

File: Documentation/docs/learn/third_party_applications.md [hunk 5]
```diff
@@ -0,0 +1,100 @@
...
+
+### Utilities/helpers
+
```
Comment (thewtex) [inline Documentation/docs/learn/third_party_applications.md:91]: In terms of Matlab interfaces, this is the most relevant now -- it does not require building and it exposes all the functionality in ITK's Python packages and remote module Python packages.

File: Documentation/docs/learn/third_party_applications.md [hunk 6]
```diff
@@ -0,0 +1,100 @@
...
+- [**MATITK**](https://www.mathworks.com/matlabcentral/fileexchange/12297-matitk)
+  was a MATLAB interface to ITK. See also on [MW File
+  exchange](http://www.mathworks.com/matlabcentral/fileexchange/12297)
+- [**MATLAB ITK Import**](https://github.com/thewtex/matlab-itk-import/tree/master)
+  allows to import any file supported by ITK into The Mathworks MATLAB. See also
+  the [CMake MatlabMex](https://gitlab.kitware.com/cmake/community/-/wikis/doc/tutorials/MatlabMex)
+  documentation.
+- [**Insight Applications**](https://itk.org/Wiki/ITK/Insight_Applications)
+- [**Command line wrappers of ITK functionality**](https://itk.org/Wiki/ITK/CommandLineTools)
```
Comment (thewtex) [inline Documentation/docs/learn/third_party_applications.md:100]: In terms of ITK command line tool, notable, maintained ones to point are the c3d command line tools developed by @pyushkevich : http://www.itksnap.org/pmwiki/pmwiki.php?n=Documentation.CommandLine

and the CLI applications developed by @aylward in the ITKTubeTK repository: https://github.com/InsightSoftwareConsortium/ITKTubeTK .

**Review by thewtex — CHANGES_REQUESTED** [review]
@jhlegarreta excellent! Thank you for this awesome DOC: work!

A few suggestions inline.

File: Documentation/docs/learn/third_party_applications.md [hunk 7]
```diff
@@ -0,0 +1,100 @@
...
+### Utilities/helpers
+
+- [**MATITK**](https://www.mathworks.com/matlabcentral/fileexchange/12297-matitk)
+  was a MATLAB interface to ITK. See also on [MW File
+  exchange](http://www.mathworks.com/matlabcentral/fileexchange/12297)
+- [**MATLAB ITK Import**](https://github.com/thewtex/matlab-itk-import/tree/master)
+  allows to import any file supported by ITK into The Mathworks MATLAB. See also
+  the [CMake MatlabMex](https://gitlab.kitware.com/cmake/community/-/wikis/doc/tutorials/MatlabMex)
+  documentation.
+- [**Insight Applications**](https://itk.org/Wiki/ITK/Insight_Applications)
```
[the remark this answers is not in the document]
Comment (jhlegarreta) [reply Documentation/docs/learn/third_party_applications.md]: Removed in 8d9650f.

File: Documentation/docs/learn/third_party_applications.md [hunk 8]
```diff
@@ -0,0 +1,100 @@
...
+- [**Analyze**](https://analyzedirect.com/) software from
+  Mayo Clinic BIR.
+- [**BioImage Suite**](https://bioimagesuiteweb.github.io/webapp/) integrated image
+  analysis software suite developed at Yale University.
+- [**BioimageXD**](https://www.bioimagexd.net/) is a free open-source
+  software package for analyzing, processing and visualizing
+  multi-dimensional microscopy images.
+- [**GoFigure2**](https://gofigure2.sourceforge.net/) is a
+  cross-platform, free open-source software (FOSS), for visualizing,
+  processing and analysing of bioimages.
```
[the remark this answers is not in the document]
Comment (jhlegarreta) [reply Documentation/docs/learn/third_party_applications.md]: Removed in 8d9650f.

File: Documentation/docs/learn/third_party_applications.md [hunk 9]
```diff
@@ -0,0 +1,100 @@
+# Third-party applications
+
+This page lists all the applications that are known to use the Insight
+Toolkit. Some of these applications are not maintained, but are kept in
+this list for informative and historical purposes.
```
[the remark this answers is not in the document]
Comment (jhlegarreta) [reply Documentation/docs/learn/third_party_applications.md]: Removed in 8d9650f.

File: Documentation/docs/learn/third_party_applications.md [hunk 10]
```diff
@@ -0,0 +1,100 @@
...
+  visualisation algorithm implementations.
+- [**MeVisLab**](https://www.mevislab.de/) is a modular framework
+  for image processing research and development with a special
+  focus on medical imaging.
+- [**SCIRun**](https://www.sci.utah.edu/cibc-software/scirun.html) is a
+  Problem Solving Environment (PSE), for modeling, simulation and
+  visualization of scientific problems.
+- [VisTrails**](https://www.vistrails.org//index.php/Main_Page) is an
+  open-source scientific workflow and provenance management system
+  that supports data exploration and visualization.
```
[the remark this answers is not in the document]
Comment (jhlegarreta) [reply Documentation/docs/learn/third_party_applications.md]: Removed in 8d9650f.

File: Documentation/docs/learn/third_party_applications.md (hunk 1 above)
Comment (jhlegarreta) [reply Documentation/docs/learn/third_party_applications.md]: Changed in 8d9650f.

File: Documentation/docs/learn/third_party_applications.md (hunk 2 above)
Comment (jhlegarreta) [reply Documentation/docs/learn/third_party_applications.md]: Changed in 8d9650f.

File: Documentation/docs/learn/third_party_applications.md (hunk 4 above)
Comment (jhlegarreta) [reply Documentation/docs/learn/third_party_applications.md]: Removed in 8d9650f.

File: Documentation/docs/learn/third_party_applications.md (hunk 5 above)
Comment (jhlegarreta) [reply Documentation/docs/learn/third_party_applications.md:83]: Changed in 8d9650f.

File: Documentation/docs/learn/third_party_applications.md (hunk 6 above)
Comment (jhlegarreta) [reply Documentation/docs/learn/third_party_applications.md]: Added/changed in 8d9650f.

File: Documentation/docs/learn/third_party_applications.md (hunk 3 above)
Comment (jhlegarreta) [reply Documentation/docs/learn/third_party_applications.md]: Removed in 8d9650f.

**Comment by hjmjohnson** [comment]
@jhlegarreta merge conflicts with other work recently committed.

**Review by thewtex — APPROVED** [review]
Of course, there are many more that could be added in the future. One that comes to mind at the moment is ANTs.

**Comment by jhlegarreta** [comment]
> @jhlegarreta merge conflicts with other work recently committed.

@hjmjohnson Solved with the latest rebase & forced push.

**Pull request merged**